systemtests/src/main/java/io/enmasse/systemtest/logs/GlobalLogCollector.java

```diff
@@ -346,6 +346,10 @@ public static void saveInfraState(Path path, String infraNamespace, ThrowableRun
             Files.writeString(path.resolve("events.txt"), KubeCMDClient.getAllEvents().getStdOut());
             Files.writeString(path.resolve("describe_nodes.txt"), KubeCMDClient.describeNodes().getStdOut());
 
+            // Investigating sporadic upgrade issue.
+            Files.writeString(path.resolve("deployments.yml"), KubeCMDClient.runOnClusterWithoutLogger("get", "deployments", "-o", "yaml").getStdOut());
```

Comment:
@k-wall can you move it please into loop with all systemtests namespaces? I think it should be useful to collect these info for all systemtests namespaces. (loop is in same file line 321...)

Reply:
Addressed.